app/controllers/initiatives_controller.rb

```diff
@@ -1,11 +1,18 @@
 # frozen_string_literal: true
 
 class InitiativesController < ApplicationController
-  before_action :find_initiative, only: [:edit, :update, :destroy]
+  before_action :find_initiative, only: [:edit, :show, :update, :destroy]
   before_action :authenticate_user!
+  before_action :populate_initiative_sets, only: [:index, :team_members_initiatives, :update]
```

Comment:
What happens if you remove `:update` from this list?

Reply:
controller test failed.